src/runtime/virtcontainers/pkg/cloud-hypervisor/Makefile

```diff
@@ -13,7 +13,7 @@ YQ := $(shell command -v yq 2> /dev/null)
 generate-client-code: clean-generated-code
 	docker run --rm \
 		--user $$(id -u):$$(id -g) \
-		-v $${PWD}:/local openapitools/openapi-generator-cli generate \
+		-v $${PWD}:/local openapitools/openapi-generator-cli:v4.3.1 generate \
```

Comment:
@devimc Thanks for finding and fixing the issue. 

Note: The `v4.3.1` tag is the latest stable release of the `openapi-generator-cli` container. This patch re-generated the clh client code with the tagged container img, which should have no functional changes on the generated client code.

/cc @jcvenegas